BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m

```diff
@@ -24,31 +24,33 @@ + (UIImage *)box_iconForItem:(BOXItem *)item
         if (item.isFolder) {
             BOXFolder *folder = (BOXFolder *)item;
             if (folder.isExternallyOwned == BOXAPIBooleanYES) {
-                icon = [UIImage imageNamed:@"icon-folder-external"
+                icon = [UIImage imageNamed:@"External"
```

Comment:
not sure to understand the question. 
This commit brings the new filetype icons that desin gave us.